components/UserMenu.tsx

```diff
@@ -120,60 +121,69 @@ const UserMenu = ({ onConnect, onDisconnect }: UserMenuProps): JSX.Element => {
                   </div>
                 )}
               </div>
-              <div>
-                <Menu.Button className="max-w-xs flex items-center text-sm rounded-full focus:outline-none">
-                  <span className="sr-only">Open user menu</span>
-                  <CogIcon
-                    className={classNames(
-                      open ? 'fill-white' : '',
-                      'h-6 w-6 md:h-5 md:w-5 fill-n-100 hover:fill-n-200'
-                    )}
-                    aria-hidden="true"
-                  />
-                </Menu.Button>
-              </div>
-              <Transition
-                as={Fragment}
-                enter="transition ease-out duration-100"
-                enterFrom="transform opacity-0 scale-95"
-                enterTo="transform opacity-100 scale-100"
-                leave="transition ease-in duration-75"
-                leaveFrom="transform opacity-100 scale-100"
-                leaveTo="transform opacity-0 scale-95"
-              >
-                <Menu.Items className="origin-bottom-right absolute right-0 bottom-12 mb-4 w-40 rounded-md shadow-lg bg-white divide-y-2 divide-zinc-50 ring-1 ring-black ring-opacity-5 focus:outline-none">
-                  <div className="px-1 py-1 ">
-                    <Menu.Item>
-                      {({ active }) => (
-                        <a
-                          onClick={onClickCopy}
-                          className={classNames(
-                            active ? 'bg-zinc-50' : '',
-                            'block rounded-md px-2 py-2 text-sm text-n-600 text-right font-normal cursor-pointer'
-                          )}
-                        >
-                          Copy wallet address
-                        </a>
-                      )}
-                    </Menu.Item>
-                  </div>
-                  <div className="px-1 py-1 ">
-                    <Menu.Item>
-                      {({ active }) => (
-                        <a
-                          onClick={onDisconnect}
-                          className={classNames(
-                            active ? 'bg-zinc-50 cursor-pointer' : '',
-                            'block rounded-md px-2 py-2 text-sm text-l-300 text-right font-semibold'
-                          )}
-                        >
-                          Disconnect wallet
-                        </a>
+              <div className="flex items-center">
+                {getEnv() === 'dev' && (
```

Comment:
Might be useful to pull this logic out into its own `isDev()` helper? I imagine this won't be the only time we need it and seems better to consolidate this kind of string matching to one place in case we ever have to modify it. I didn't see any other instances of it yet though so not pressing!